sympy/series/fourier.py

```diff
@@ -440,16 +438,212 @@ def __sub__(self, other):
         return self.__add__(-other)
 
 
-class FiniteFourierSeries(Basic):
-    def __new__(cls, *args):
-        obj = Basic.__new__(cls, *args)
-        return obj
+class FiniteFourierSeries(FourierSeries):
+    r"""Represents Finite Fourier sine/cosine series.
 
-    def truncate(self, n=3):
-        return Add(*self._args)
+       For how to compute Fourier series, see the :func:`fourier_series`
```

Comment:
Is this indentation correct? Should it be 4 spaces as on the first docstring line 442.

Reply:
It's part of the docstring. Should I change it?

Reply:
I think that the indentation is usually expected to be the same although Python does not check it inside a triple quote string.